2025-05-13 03:12:40,965 - INFO - URL OK: http://hls1.sua.tv/live/globotvbahiafhdbr2/s.m3u8
2025-05-13 03:12:55,981 - ERROR - Request Error http://89.45.12.66:8080/live/toptv_207670/LgNXVm3I/14161.m3u8: HTTPConnectionPool(host='89.45.12.66', port=8080): Max retries exceeded with url: /live/toptv_207670/LgNXVm3I/14161.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d4d254c20>, 'Connection to 89.45.12.66 timed out. (connect timeout=15)'))
2025-05-13 03:13:10,997 - ERROR - Request Error http://190.102.43.113/Formigatv/offilinetv2806/1541684.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/1541684.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d4d255100>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-13 03:13:26,014 - ERROR - Request Error http://190.102.43.113/Formigatv/offilinetv2806/1541685.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/1541685.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d4d2559d0>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-13 03:13:26,079 - ERROR - Request Error http://15.235.15.230:14785/: HTTPConnectionPool(host='15.235.15.230', port=14785): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d256300>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:13:26,145 - ERROR - Request Error http://15.235.15.230:14757/: HTTPConnectionPool(host='15.235.15.230', port=14757): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d256c00>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:13:26,211 - ERROR - Request Error http://15.235.15.230:14517/: HTTPConnectionPool(host='15.235.15.230', port=14517): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d256060>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:13:26,276 - ERROR - Request Error http://15.235.15.230:14519/: HTTPConnectionPool(host='15.235.15.230', port=14519): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d255490>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:13:26,343 - ERROR - Request Error http://15.235.15.230:14488/: HTTPConnectionPool(host='15.235.15.230', port=14488): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d254290>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:13:26,411 - ERROR - Request Error http://15.235.15.230:14464/: HTTPConnectionPool(host='15.235.15.230', port=14464): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d256f00>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:13:26,477 - ERROR - Request Error http://15.235.15.230:14462/: HTTPConnectionPool(host='15.235.15.230', port=14462): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d257860>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:13:41,493 - ERROR - Request Error http://89.45.12.66:8080/live/toptv_207670/LgNXVm3I/14146.m3u8: HTTPConnectionPool(host='89.45.12.66', port=8080): Max retries exceeded with url: /live/toptv_207670/LgNXVm3I/14146.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d4d264230>, 'Connection to 89.45.12.66 timed out. (connect timeout=15)'))
2025-05-13 03:13:56,509 - ERROR - Request Error http://89.45.12.66:8080/live/toptv_207670/LgNXVm3I/36283.m3u8: HTTPConnectionPool(host='89.45.12.66', port=8080): Max retries exceeded with url: /live/toptv_207670/LgNXVm3I/36283.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d4d2570e0>, 'Connection to 89.45.12.66 timed out. (connect timeout=15)'))
2025-05-13 03:14:11,515 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/1541727.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/1541727.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d4d254aa0>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-13 03:14:26,529 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320566.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320566.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d4d2544d0>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-13 03:14:41,546 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320567.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320567.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d4d255cd0>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-13 03:14:56,559 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320570.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320570.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d4d2644a0>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-13 03:15:11,575 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320569.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320569.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d4d264e00>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-13 03:15:26,583 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320563.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320563.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d4d255d30>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-13 03:15:41,595 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320564.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320564.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d4d254f50>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-13 03:15:42,328 - WARNING - URL Error https://cdn-3.nxplay.com.br/GLOBO_SP_TK/index.m3u8: Status Code 403
2025-05-13 03:15:42,755 - INFO - URL OK: http://hls1.sua.tv:80/live/globosphd/s.m3u8
2025-05-13 03:15:42,955 - WARNING - URL Error http://xcui.vitvabc.xyz:8880/laok300/8pEZjdeMPad7/32370: Status Code 401
2025-05-13 03:15:43,114 - WARNING - URL Error http://xcui.vitvabc.xyz:8880/laok300/8pEZjdeMPad7/32266: Status Code 401
2025-05-13 03:15:59,146 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112659
2025-05-13 03:16:16,314 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112660
2025-05-13 03:16:33,324 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112664
2025-05-13 03:16:49,174 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112665
2025-05-13 03:16:50,005 - WARNING - URL Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112666: Status Code 403
2025-05-13 03:17:16,758 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112667: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-13 03:17:34,486 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112668: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-13 03:17:52,206 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112669: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-13 03:18:09,784 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112670: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-13 03:18:25,772 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112671: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-13 03:18:43,638 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112672: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-13 03:18:59,734 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112673: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-13 03:19:16,280 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112674
2025-05-13 03:19:32,289 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112675
2025-05-13 03:19:47,299 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112676
2025-05-13 03:20:04,311 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112677
2025-05-13 03:20:19,831 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112678
2025-05-13 03:20:36,338 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112679
2025-05-13 03:20:53,350 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112680
2025-05-13 03:21:10,359 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112681
2025-05-13 03:21:27,435 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112682
2025-05-13 03:21:43,379 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112683
2025-05-13 03:21:44,201 - WARNING - URL Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112684: Status Code 403
2025-05-13 03:22:12,070 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112685: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-13 03:22:29,798 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112686: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-13 03:22:47,672 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112687: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-13 03:23:03,655 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112688: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-13 03:23:19,641 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112689: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-13 03:24:02,474 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112690
2025-05-13 03:24:31,214 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112691: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-13 03:24:48,781 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112692: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-13 03:25:04,761 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112693: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-13 03:25:20,749 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112694: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-13 03:25:20,818 - ERROR - Request Error http://15.235.15.230:14783/: HTTPConnectionPool(host='15.235.15.230', port=14783): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d265310>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:20,887 - ERROR - Request Error http://15.235.15.230:14478/: HTTPConnectionPool(host='15.235.15.230', port=14478): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d256f00>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:20,955 - ERROR - Request Error http://15.235.15.230:14492/: HTTPConnectionPool(host='15.235.15.230', port=14492): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d254560>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:21,024 - ERROR - Request Error http://15.235.15.230:14485/: HTTPConnectionPool(host='15.235.15.230', port=14485): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d255be0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:21,092 - ERROR - Request Error http://15.235.15.230:14495/: HTTPConnectionPool(host='15.235.15.230', port=14495): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d256780>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:21,161 - ERROR - Request Error http://15.235.15.230:14502/: HTTPConnectionPool(host='15.235.15.230', port=14502): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d2577a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:21,230 - ERROR - Request Error http://15.235.15.230:14506/: HTTPConnectionPool(host='15.235.15.230', port=14506): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d2671d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:21,298 - ERROR - Request Error http://15.235.15.230:14504/: HTTPConnectionPool(host='15.235.15.230', port=14504): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d257da0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:21,367 - ERROR - Request Error http://15.235.15.230:14508/: HTTPConnectionPool(host='15.235.15.230', port=14508): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d257470>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:21,437 - ERROR - Request Error http://15.235.15.230:14513/: HTTPConnectionPool(host='15.235.15.230', port=14513): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d2557f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:21,505 - ERROR - Request Error http://15.235.15.230:14512/: HTTPConnectionPool(host='15.235.15.230', port=14512): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d254230>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:21,573 - ERROR - Request Error http://15.235.15.230:14510/: HTTPConnectionPool(host='15.235.15.230', port=14510): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d254dd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:21,642 - ERROR - Request Error http://15.235.15.230:14516/: HTTPConnectionPool(host='15.235.15.230', port=14516): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d265cd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:21,711 - ERROR - Request Error http://15.235.15.230:14492/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14492): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d365fd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:21,780 - ERROR - Request Error http://15.235.15.230:14487/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14487): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d254c50>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:21,848 - ERROR - Request Error http://15.235.15.230:14499/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14499): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d257650>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:21,916 - ERROR - Request Error http://15.235.15.230:14513/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14513): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d256660>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:21,984 - ERROR - Request Error http://15.235.15.230:14513/: HTTPConnectionPool(host='15.235.15.230', port=14513): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d257aa0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:22,412 - INFO - URL OK: http://hls1.sua.tv/live/globotvbahiafhdbr2/s.m3u8
2025-05-13 03:25:22,767 - WARNING - URL Error http://s1.1-cup.net:2095/solucao111hgfgh77hu78/xplus1111XXhjgyu87/51: Status Code 404
2025-05-13 03:25:22,835 - ERROR - Request Error http://15.235.15.230:14487/: HTTPConnectionPool(host='15.235.15.230', port=14487): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d4d265bb0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-13 03:25:23,576 - WARNING - URL Error https://cdn-3.nxplay.com.br/GLOBO_SP_TK/tracks-v1a1/mono.m3u8: Status Code 403
2025-05-13 03:25:24,004 - INFO - URL OK: http://hls1.sua.tv/live/globotvbahiafhdbr2/s.m3u8
2025-05-13 03:25:24,386 - INFO - URL OK: http://hls1.sua.tv:80/live/globosphd/s.m3u8
2025-05-13 03:25:24,639 - WARNING - URL Error http://metv.fun:80/4x6cXF/JDqRqj/90059.m3u8: Status Code 404
2025-05-13 03:25:24,840 - WARNING - URL Error http://metv.fun:80/4x6cXF/JDqRqj/91431.m3u8: Status Code 404
2025-05-13 03:25:25,063 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/90034.m3u8: Status Code 404
2025-05-13 03:25:25,225 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91421.m3u8: Status Code 404
2025-05-13 03:25:25,413 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91435.m3u8: Status Code 404
2025-05-13 03:25:25,587 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91463.m3u8: Status Code 404
2025-05-13 03:25:25,777 - WARNING - URL Error http://metv.fun:80/4x6cXF/JDqRqj/90064.m3u8: Status Code 404
2025-05-13 03:25:25,952 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/90019.m3u8: Status Code 404
2025-05-13 03:25:26,130 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/95441.m3u8: Status Code 404
2025-05-13 03:25:26,312 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91405.m3u8: Status Code 404
2025-05-13 03:25:26,483 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91413.m3u8: Status Code 404
2025-05-13 03:25:26,645 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/120082.m3u8: Status Code 404
2025-05-13 03:25:26,832 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91443.m3u8: Status Code 404
2025-05-13 03:25:27,005 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91445.m3u8: Status Code 404
2025-05-13 03:25:27,192 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/95483.m3u8: Status Code 404
2025-05-13 03:25:27,368 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/97396.m3u8: Status Code 404
2025-05-13 03:25:27,582 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91456.m3u8: Status Code 404
2025-05-13 03:25:27,768 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91460.m3u8: Status Code 404
2025-05-13 03:25:27,931 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/95448.m3u8: Status Code 404
2025-05-13 03:25:28,083 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91461.m3u8: Status Code 404
2025-05-13 03:25:28,261 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91467.m3u8: Status Code 404
2025-05-13 03:25:28,454 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91470.m3u8: Status Code 404
2025-05-13 03:25:28,628 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91471.m3u8: Status Code 404
